pytripgui/add_vois_vc/add_vois_cont.py

```diff
@@ -0,0 +1,90 @@
+from pytrip.vdx import create_sphere, create_cube
+
+import logging
+
+from pytripgui.add_vois_vc.add_voi_vc.add_voi_cont import AddVOIController
+from pytripgui.add_vois_vc.add_voi_vc.add_voi_view import AddVOIQtView
+from pytripgui.add_vois_vc.voi_widget import SphericalVOIWidget, CuboidalVOIWidget
+
+logger = logging.getLogger(__name__)
+
+
+class AddVOIsController:
+    def __init__(self, model, view):
+        self.model = model
+        self.view = view
+        self.is_accepted = False
+        self._setup_callbacks()
+
+        self._set_view_from_model()
+
+    def _setup_callbacks(self):
+        self.view.accept_buttons.accepted.disconnect()
+        self.view.accept_buttons.accepted.connect(self._save_and_exit)
+
+        self.view.add_voi_button.emit_on_click(lambda: self._create_add_voi_dialog())
```

Comment:
Replace lambda with function name

Reply:
I'm trying to figure it out why, but in this instance it has to be a verbatim lambda or else it doesn't work.

Reply:
```suggestion
        self.view.add_voi_button.emit_on_click(self._create_add_voi_dialog)
```

Will it work like that ?

Reply:
Nope